samples/drivers/uart/native_tty/README.rst

```diff
@@ -0,0 +1,95 @@
+.. _sample-uart-native-tty:
+
+Native TTY UART
+###############
+
+Overview
+********
+
+This sample demonstrates the usage of Native TTY UART API. By using two UART to
+USB bridge dongles that are wired together sample writes data to one dongle and
+reads it from the another.
+
+The source code for this sample application can be found at:
+:zephyr_file:`samples/drivers/uart/native-tty`.
+
+You can learn more about usage of Native TTY UART driver in
+:ref:`the Serial Ports <serial_ports>` section of the Native posix board
+documentation.
+
+Requirements
+************
+
+#. Two UART to USB bridge dongles. Each dongle must have its TX pin wired to the
+   other dongle's RX pin. The ground connection is not needed. Both dongles must
+   be plugged into the host machine.
+#. Sample expects that dongles appear as ``/dev/ttyUSB0`` and ``/dev/ttyUSB1``
+   in the system. You can check for them with ``ls -l /dev/tty*`` command. If
```

Comment:
Maybe better?
"You can check what they are in your system by running the command ``ls -l /dev/tty*``"